README.md

```diff
@@ -39,6 +39,7 @@ Please note that on the interface, the Redis server info button will not work. F
 * `REDIS_USE_TLS` - enable TLS true or false (`false` by default)
 * `REDIS_USER` - user to connect to redis (no user by default)
 * `REDIS_PASSWORD` - password to connect to redis (no password by default)
+* `REDIS_FAMILY` - either 4 or 6, defaults to 4
```

Comment:
In accordance with the documentation:
https://github.com/redis/node-redis/blob/master/docs/client-configuration.md
```suggestion
* `REDIS_FAMILY` - IP Stack version (one of 4 | 6 | 0) (`0` by default)
```